eventbus/src/main/java/pl/edu/uj/kimage/eventbus/VertXEventBus.java

```diff
@@ -35,10 +35,14 @@ public void registerEventListener(final Class<? extends Event> eventClass, Event
 
     @Override
     public void registerCommandHandler(Class<? extends Command> commandClass, CommandHandler commandHandler) {
-        registerEventListener(commandClass, (EventListener) commandHandler);
+        eventBus.consumer(toTopicName(commandClass))
+                .handler(objectMessage -> {
+                    Command command = messageTranslator.deserialize(commandClass, (String) objectMessage.body());
+                    commandHandler.receive(command);
+                });
     }
 
-    private String toTopicName(Class<? extends Event> eventClass) {
+    private String toTopicName(Class eventClass) {
         return eventClass.getCanonicalName();
```

Comment:
this method is once used by Command implementations and once by Event implementations. Is it correct usage? I ask, because the parameter name says something else